modules/s3/s3import.py

```diff
@@ -3935,6 +3936,24 @@ def parseACL(_acl):
                             *acls[rulelist[0]])
 
     # -------------------------------------------------------------------------
+    def import_font(self, url):
```

Comment:
There are definitely other places that use zipfile.  Don't see other uses of tarfile.  There is also a gzip package, but a comment where that is used indicates it may not be always available, where tarfile does seem to be included with Python (or at least, I did not need to install it).